specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/AscOperations_Get.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "613192d7-503f-477a-9cfe-4efc3ee2bd60",
+    "location": "West US",
```

Comment:
Should this be "westus"? I've never seen "West US" in Azure REST results.

Reply:
https://docs.microsoft.com/en-us/rest/api/redis/redis/create#examples
https://docs.microsoft.com/en-us/rest/api/container-service/containerservices/createorupdate#examples
I see a mix of inputs here.  Including West US.  
While creating a VM, it has (US) West US as an input region in the selection box.
Not sure.

Reply:
In the REST context, I always see it as `westus`